tests/monotouch-test/CoreServices/FSEventStreamTest.cs

```diff
@@ -221,13 +231,18 @@ void CreateFilesAndWaitForFSEventsThread ()
 						throw new TimeoutException (
 							$"test has timed out at {s_testTimeout.TotalSeconds}s; " +
 							"increase the timeout or reduce the number of files created. " +
-							$"Created directories: {createdDirCount} Created files: {createdFileCount} Removed files: {removedFileCount} Created then removed files: {createdThenRemovedFileCount}");
+							$"Created directories: {createdDirCount} (exit condition: 0) Created files: {createdFileCount} (exit condition: 0) Removed files: {removedFileCount} Created then removed files: {createdThenRemovedFileCount}\n{log.Count} Log lines:\n\t{string.Join ("\n\t", log)}");
 				}
 			}
 
+			List<string> log = new List<string> ();
```

Comment:
very nit but declaration could be a bit simpler
```suggestion
			List<string> log = new ();
```